### PR TITLE
Service methods should be possible to specify its error type 

### PR DESCRIPTION
**Pull request by admire93**
https://github.com/spoqa/nirum/issues/38

### CODE REVIEW

File: examples/pdf-service.nrm [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@error("")
```
Comment (dahlia) [inline examples/pdf-service.nrm]: Annotations don't have to have arguments (including parentheses).  Therefore we can simply this as `@error`.  See also the [parser code][1].

[1]: https://github.com/spoqa/nirum/blob/cb59c832dd7f6f09debdcec6934ecc80fb223e14/src/Nirum/Parser.hs#L161-L168

**Comment by admire93** [comment]
이제 남은것은 Runtime쪽에서 해결하는정도입니다 리뷰부탁드려요 @dahlia

**Comment by admire93** [comment]
@dahlia 리뷰를 부탁드립니다.

File: src/Nirum/Targets/Python.hs [hunk 2]
```diff
@@ -693,10 +694,12 @@ class $className(object):
     def __hash__(self){ret "int"}:
         return hash(($hashText,))
 |]
-compileTypeDeclaration src d@TypeDeclaration { typename = typename'
-                                             , type' = UnionType tags
-                                             } = do
+compileTypeDeclaration src
+                       TypeDeclaration { typename = typename'
+                                       , type' = UnionType tags
+                                       , typeAnnotations = annotations } = do
```
Comment (dahlia) [inline src/Nirum/Targets/Python.hs]: 닫는 중괄호를 여는 중괄호에 맞춰서 들여쓰기 부탁합니다.

**Comment by admire93** [comment]
https://github.com/spoqa/nirum-python/issues/71 is folloing issue.

**Pull request merged**